core/client/fs/src/main/java/alluxio/client/file/cache/store/PageStoreOptions.java

```diff
@@ -48,4 +63,46 @@ public void setRootDir(String rootDir) {
   public String getRootDir() {
     return mRootDir;
   }
+
+  /**
+   * @return the size of the page
+   */
+  public long getPageSize() {
+    return mPageSize;
+  }
+
+  /**
+   * @param pageSize the size of the page
```

Comment:
nit "pageSize the size of the page in bytes" (is it?)

Reply:
Updated.